### ISSUE TITLE
it seems that python 3.8 doesn't work

### ISSUE THREAD

**Issue by maxi2015qwe** [stardist/stardist#152]
## ❓ Questions and Help
How can i accse my own training model ?
---------------------------------------------------------------------------
I have completed my own training model and I want to call my model using a python script on pycharm.

I read your description of the program and Only the pre-trained model is called.

**Comment by uschmidt83** [comment]
Sorry, I don't understand the issue. Also, please use the image.sc forum (as [instructed](https://github.com/stardist/stardist/issues/new?assignees=&labels=&template=_1_question-help-support.md&title=) when you opened the issue here).

**Comment by maxi2015qwe** [comment]
thanks for your Replay .I got my training Model ,so i tryed to close my
question. but today i used my Model to predicted the Image but the result
is not so good.

My images are not all elliptical, sometimes they are bumpy. But the
Prediction is all elliptical. Maybe it is because of the n_rays  too small.

In this case which parameter should i change?

and now i changed the n_rays = 64  and the train_batch_size=64 and it takes
me so long time.

so I want to use gpu to train 2d models
I have downloaded tensorflow and Gputools according to the instructions but
I still can't compute with Gpu.

If I use the code you showed  in the example, which parameters should I
adjust in order to use Gpu?



Uwe Schmidt ***@***.***> 于2021年7月19日周一 下午3:29写道：

**Comment by uschmidt83** [comment]
Again, this is not the place to discuss this. Please follow this (which I've copied from the [Troubleshooting & Support](https://github.com/stardist/stardist/blob/master/README.md#troubleshooting--support) section of our README):

1. Please first take a look at the [frequently asked questions (FAQ)]( https://stardist.net/docs/faq.html).
2. If you need further help, please go to the [image.sc forum](https://forum.image.sc) and try to find out if the issue you're having has already been discussed or solved by other people. If not, feel free to create a new topic there and make sure to use the tag `stardist` (we are monitoring all questions with this tag).
3. If you have a technical question related to the source code or believe to have found a bug, feel free to [open an issue](https://github.com/stardist/stardist/issues), but please check first if someone already created a similar issue.
